src/NuGetRepack/Common/NuGetVersionUpdater.cs

```diff
@@ -215,6 +228,13 @@ private static void LoadPackages(IEnumerable<string> packagePaths, Dictionary<st
                         }
                     }
 
+                    if (isDotnetTool)
```

Comment:
It will has some waste compute but the alternative of break 2 layer of look is goto...